### PR TITLE
support l2 boosts

### PR DESCRIPTION
**Pull request by mendesfabio**

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -272,4 +302,7 @@ type User @entity {
 
   " List of locks the user created "
   votingLocks: [VotingEscrowLock!] @derivedFrom(field: "user")
+
+  " List of omni locks the user created "
+  omniVotingLocks: [VotingEscrowLock!] @derivedFrom(field: "user")
```
Comment (markusbkoch) [inline schema.graphql:299]: Shouldn't this be `[OmniVotingEscrowLock!]`

File: src/voting.ts [hunk 2]
```diff
@@ -117,23 +89,52 @@ export function handleSupply(event: Supply): void {
...
-  setOmniVotingEscrowLock(
-    event.address,
-    event.params.localUser,
-    'To',
-    event.params.dstChainId,
-    event.params.userPoint.bias,
-    event.params.userPoint.slope,
-  );
+  // TODO: find diff between localUser and remoteUser
+  // maybe create 2 separate fields for this
```
Comment (markusbkoch) [inline src/voting.ts:119]: Yes, agree we should store `localUser` and `remoteUser` in the `OmniVotingEscrowLock` entity

Comment (mendesfabio) [reply src/voting.ts:119]: I made only `localUser` a reference to `User` entity and left `remoteUser` as a bytes field - wdyt? https://github.com/balancer/gauges-subgraph/pull/34/commits/183c3b8e567c5e77ff125b91107b44dcb4c60bbc

**Comment by mendesfabio** [comment]
opened #42 for us to merge first only the veBAL snapshots.

**Pull request merged**